tests/acceptance/acceptancesuites/filtersearchsuite.js

```diff
@@ -32,63 +25,39 @@ test(`filtersearch works with back/forward navigation and page refresh`, async t
   };
   const filterTags = Selector('.yxt-ResultsHeader-removableFilterTag');
   const filterSearch = FacetsPage.getFilterSearch();
-  const virginiaFilter = {
```

Comment:
removed these because they were being inconsistent on circleci, I checked the browserstack video and it looked like the correct filtersearch option was being selected, but the actual filter being sent in the request was neither the virginia nor new york filter (and also this was passing locally) which makes me thinks circleci is getting a different filtersearch response
![image](https://user-images.githubusercontent.com/23005393/110867774-57d13b00-8295-11eb-958b-59b6dd0d49cb.png)